site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,273 @@
+# Cross namespace references and ReferencePolicy
+
+In the Gateway API, it is possible to have references between objects cross namespace boundaries.
+In particular, it's expected that Gateways and Routes may exist in different namespaces,
+or that Services may be referred to by Routes in a another namespace.
+
+However, this significantly violates the idea of a namespace as the edge of a trust domain.
+In order to bring cross-namespace references under the control of the owner of the referent object's namespace,
+the Gateway API has a ReferencePolicy object that must be created in the referent namespace for the reference to be successful.
+
+To put this another way, if an object is referred to from outside its namespace,
+the object's owner must create a ReferencePolicy object that describes how that reference is allowed.
+This page explains how this process works.
+
+In the past, we've seen that forwarding traffic across namespace boundaries is a desired feature,
+but without the kinds of safeguards proposed here, [vulnerabilities](https://github.com/kubernetes/kubernetes/issues/103675)
+can emerge.
```

Comment:
Maybe start with this paragraph to build up to the problem we are trying to solve and then propose the solution.